src/components/canvas-tools/tool-tile.tsx

```diff
@@ -38,6 +38,7 @@ interface IProps {
   context: string;
   docId: string;
   scale?: number;
+  tabIndex?: number;
```

Comment:
Is the idea that `tabIndex` is necessary for keyboard focus to do things like backspace points?

Also seems useful in some world where we get to think about accessible controls

Reply:
Yes, calling `focus()` for a `<div>` element is a no-op unless it has a non-empty `tabIndex`.